src/tests/unit/test_slide.py

```diff
@@ -71,24 +72,24 @@ def it_knows_its_wsi_name(self, wsi_name_fixture):
     def it_knows_its_scaled_image_path(self, scaled_img_path_fixture, resampled_dims_):
         wsi_path, proc_path, wsi_dims, expected_value = scaled_img_path_fixture
         resampled_dims_.return_value = wsi_dims
-        slide = Slide(wsi_path, proc_path, scale_factor=22)
+        slide = Slide(wsi_path, proc_path)
 
-        scaled_img_path = slide.scaled_image_path
+        scaled_img_path = slide.scaled_image_path(22)
```

Comment:
maybe here is better to pass the scale factor in kw mode `scale_factor=22`, i think it's more meaningful and significative for who read the tests